src/CssToInlineStyles.php

```diff
@@ -197,6 +202,35 @@ protected function inline(\DOMDocument $document, array $rules)
         return $document;
     }
 
+    /**
+     * Appends a style tag to the body element containing all media queries.
+     *
+     * @param \DOMDocument $document
+     * @param string[] $mediaQueries
+     *
+     * @return \DOMDocument
+     */
+    protected function appendMediaQueries(\DOMDocument $document, array $mediaQueries)
+    {
+        if (!count($mediaQueries)) {
+            return $document;
+        }
+        $style = $document->createElement('style', implode($mediaQueries));
+        if ($style) {
+            $head = $document->getElementsByTagName('head')->item(0);
+            // Partials aren't supported (missing head tag). But lets play nice.
+            if (null !== $head) {
+                $head->appendChild($style);
+            } else {
+                $docNode = $document->documentElement;
+                $head = $document->createElement('head');
+                $head->appendChild($style);
+                $docNode->insertBefore($head, $docNode->firstChild);
```

Comment:
instead this broken if the HML has a `<html>` tag without a `<head>` tag ? An absent `<head>` tag does not only happen in partials